.gitignore

```diff
@@ -18,6 +18,7 @@
 /npm-debug.log*
 /testem.log
 /yarn-error.log
+/.firebase/hosting.ZGlzdA.cache
```

Comment:
This won't be enough.
Take a look at
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
I got it, I will remove it with git rm --cached <file> after we finish with this merge pull request!

Reply:
You should remove now, commit, and push. Waiting for this so I can merge